Gruntfile.js

```diff
@@ -2,10 +2,42 @@ module.exports = function (grunt) {
 
     // Project configuration.
     grunt.initConfig({
-        pkg: grunt.file.readJSON('package.json')
-
+        pkg: grunt.file.readJSON('package.json'),
+        sass: {
+            options: {
+                sourcemap: 'none'
+            },
+            dist: {
+                files: {
+                    'css/blog-sass.css': 'css/sass/blog-sass.scss'
+                }
+            }
+        },
+        cssmin: {
+            options: {
+                sourceMap: true
+            },
+            target: {
+                files: {
+                    'css/blog-sass.css.min': ['css/blog-sass.css']
```

Comment:
A minor point but I think the `-sass` suffix in the `.scss` files is a bit redundant. I'd change the file source file name to `blog.scss`, the compiled to `blog.css` and the minified to `blog.min.css`

Reply:
suffix removed